walkthroughs/1-template-walkthrough/walkthrough.json

```diff
@@ -1,13 +1,180 @@
 {
   "dependencies": {
-      "managedServices": [
+    "managedServices": [
       {
         "name": "amqonline"
       },
       {
         "name": "fuse"
       }
     ],
-    "templates": []
+    "templates": [
```

Comment:
@pb82 so, this creates a template in the cluster then create the resources, or uses the *existing* postgresql template?